setup.py

```diff
@@ -252,7 +253,7 @@ def parse_setuppy_commands():
     good_commands = ('develop', 'sdist', 'build', 'build_ext', 'build_py',
                      'build_clib', 'build_scripts', 'bdist_wheel', 'bdist_rpm',
                      'bdist_wininst', 'bdist_msi', 'bdist_mpkg', 'build_src',
-                     'version')
+                     'dist_info',)
```

Comment:
I'm not convinced this is the best place for 'dist_info', but it is consistent with how it was working before.